internal/forms.py

```diff
@@ -23,6 +23,8 @@ def __init__(self, *args, **kwargs):
 
 
 class EditMemberForm(ModelForm):
+    card_number = IntegerField(label=_("Card number (EM)"))
```

Comment:
Maybe we should make a custom field and widget for EM numbers, that perform validation and have [a label](https://fomantic-ui.com/elements/input.html#labeled) on the input field that says EM.